internal_docs/review-apps.md

```diff
@@ -0,0 +1,26 @@
+# Pull Request Review apps
+
+This repo uses [Heroku review apps](https://devcenter.heroku.com/articles/github-integration-review-apps)
+to give reviewers a way to easily preview and test changes to the app.
+
+If a pull request is created from a fork of this repo (typical if the
+contributor is not part of the alphagov organisation) then the review app will
+not be created automatically; however it can be created manually from the
+Heroku dashboard.
+
+Previews of Pull Requests are automatically published to a URL which has the
+prefix `govuk-prototype-kit` followed by the identifier number of the pull request.
+
+For example, pull request #137 would be deployed to `govuk-prototype-kit-pr-137.herokuapp.com`.
+
+The govuk-design-system-ci should create a GitHub deployment as Heroku builds
+and deploys the review app. The deployment will appear both as an event in the
+PR timeline and as an active deployment in the deployment status above the PR
+status checks. Once the review app has been deployed a button linking to the
```

Comment:
```suggestion
status checks. Once the review app has been deployed, a button linking to the
```

Reply:
Do we need to say what the button enables users to do? (To open the review app and preview their changes?)